relay-server/src/actors/outcome.rs

```diff
@@ -127,6 +130,7 @@ impl Outcome {
             Outcome::RateLimited(_) => 2,
             Outcome::Invalid(_) => 3,
             Outcome::Abuse => 4,
+            Outcome::ClientDiscard(_) => 5,
```

Comment:
It appears that both the outcomes consumer in getsentry as well as snuba can deal with new values 👍